doc/pages/expansions.asciidoc

```diff
@@ -48,9 +48,9 @@ for full details). For example:
 
 * `echo "x%val{session}x"` echoes the current session ID, surrounded by `x`
 
-* `echo %{%val{session}}"` echoes the the literal text `%val{session}`
+* `echo "%{%val{session}}"` echoes the the literal text `%val{session}`
 
-* `echo %sh{ echo %val{session} }"` echoes the literal text `%val{session}`
+* `echo "%sh{ echo %val{session} }"` echoes the literal text `%val{session}`
```

Comment:
Likewise, this example is supposed to show that a single layer of %sh{} will prevent the contents from being expanded. The extra double-quote should be removed.